spritz/app/build.gradle

```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 26
+    buildToolsVersion "26.0.1"
+    defaultConfig {
+        applicationId "com.novoda.spritz.sample"
+        minSdkVersion 14
+        targetSdkVersion 26
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile project(path: ':spritz-android')
+
+    compile 'com.android.support:appcompat-v7:26.1.0'
+    compile 'com.android.support.constraint:constraint-layout:1.0.2'
+    compile 'com.android.support:design:26.1.0'
+    compile 'com.airbnb.android:lottie:2.1.0'
```

Comment:
Lottie is currently at [`2.2.5`](https://github.com/airbnb/lottie-android#download). Besides, this is coming from the library module already.

Reply:
Done in bc0c033522d78ef89f5798d780a84a1362b52a4f.